api/v1/collector_server.proto

```diff
@@ -17,12 +17,12 @@ message UsageRecord {
   int32 latency_ms = 8;
 }
 
-message CollectUsageRequest {
+message CreateUsageRequest {
   repeated UsageRecord records = 1;
 }
 
-message CollectUsageResponse {}
+message CreateUsageResponse {}
```

Comment:
If we want to stick to Google API guideline, this should return `Usage`, but I'm not sure. It looks a bit weird..

https://cloud.google.com/apis/design/standard_methods#create

Reply:
I see. It uses a different style than `buf lint.`